README.rst

```diff
@@ -111,17 +95,54 @@ The `full documentation <https://serenata_toolbox.readthedocs.io>`_ is still a w
 
   $ cd docs
   $ make clean;make rst;rm source/modules.rst;make html
-  
-Run Unit Test suite
--------------------
+
+Contributing
+------------
+
+Within your `virtualenv <https://virtualenv.pypa.io/en/stable/>`_:
+
+::
+
+  $ git clone https://github.com/datasciencebr/serenata-toolbox.git
+  $ python setup.py develop
+
+Always add tests to your contribution — if you want to test it locally before opening the PR:
 
 ::
 
   $ python -m unittest discover tests
 
-Source Code
------------
+When the tests are passing, also check for coverage of the modules you edited or added — if you want to check it before opening the PR:
+
+::
+
+  $ pip install coverage
+  $ coverage run -m unittest discover tests
+  $ coverage html
+  $ open htmlcov/index.html
+
+Follow `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ and best practices implemented by `Landscape <https://landscape.io>`_ in the `veryhigh` strictness level — if you want to check them locally before opening the PR:
```

Comment:
💅 I would change here to add a little something about line length, because even tough we are follow most of PEP8's best practice guide, we are not strict to `max line length = 80`